doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -542,6 +548,14 @@ Enabling and disabling notations
 
       Scopes are not compatible with :ref:`abbreviations <Abbreviations>`.
 
+Here are examples:
```

Comment:
```suggestion
   .. example:: Enabling and disabling notations
```

Reply:
Thanks @jfehrle.

About examples, I realize that there are other places where we could use `.. example::` for better uniformity.